nvtabular/workflow.py

```diff
@@ -85,21 +75,13 @@ def __init__(
         cat_names=None,
         cont_names=None,
         label_name=None,
-        feat_ops=None,
-        stat_ops=None,
-        df_ops=None,
-        to_cpu=True,
         config=None,
+        # TODO: move these out
```

Comment:
Just to clarify - The plan is to turn "exporting" into a separate Op in the future?

Reply:
Yes, we're planning on having separate 'export' ops in the future. The default 'export_path' does get used in the 'phases_export' function right now - though that could probably be removed too